evals/elsuite/already_said_that/scripts/make_plots.py

```diff
@@ -23,6 +23,7 @@ def zero_if_none(input_num):
     "cot/gpt-3.5-turbo",
     "gpt-3.5-turbo",
     "gpt-4-base",
+    "gpt-4o",
```

Comment:
nit: `4o-mini` instead of `4o` maybe to reduce the costs?

Reply:
I'll do a followup to add mini too, and send it upstream as well